checklist/checklist.md

```diff
@@ -0,0 +1,77 @@
+# Smoky testing
+- [ ] Is main page loaded
```

Comment:
```diff
+ [ ] Site base URL should opening
- [ ] Is registration available
- [ ] Is login available
```

@EmelyanovaQA what can you suggest here?

Reply:
I suggest to correct a little, f.ex:
- [ ] Registration form is filled with valid data 
- [ ] Authorization form is filled with valid data
I suggest to add points  "Logout user" and "Edit user settings"